Corrects the third example on Remothe-hooks.md

humansonofhuman opened the pull request:
While I was working whit an afterRemote Hook I tried to delete a property of the results following the example in https://loopback.io/doc/en/lb3/Remote-hooks#common/models/customer.js (the second one) but it didn't worked for me, so I google it, and I saw this closed issue strongloop/loopback#1816 and discovered that I needed to use result.unsetAttribute('property_name') to do it… so since the example didn't worked out I was thinking on send a pull request changing that example.

Commit:
Corrects the third example on Remothe-hooks.md

While I was working whit an afterRemote Hook I tried to delete a property of the results following the example in https://loopback.io/doc/en/lb3/Remote-hooks#common/models/customer.js (the second one) but it didn't worked for me, so I google it, and I saw this closed issue strongloop/loopback#1816 and discovered that I needed to use result.unsetAttribute('property_name') to do it… so since the example didn't worked out I was thinking on send a pull request changing that example.

Commit:
Removes delete word forgotten

Ups

dhmlau commented:
@mmgescuelaup , thanks for your PR and sorry about the delay.  

@jannyHou, reading the thread in https://github.com/strongloop/loopback/issues/1816, it was mentioned in [this comment](https://github.com/strongloop/loopback/issues/1816#issuecomment-157302514) to have an additional check.  Do you think it is necessary?  Or the change suggested in this PR should be sufficient? Thanks.

dhmlau commented:
@mmgescuelaup , could you please fix your linter error? We have the limit of 50 characters for the first line and 72 characters for the subsequent line in the commit messages. 

In case you're new to GitHub, you can amend your commit message by running `git commit --amend`. See instructions https://help.github.com/articles/changing-a-commit-message/. 

Please reference our commit message guideline for details: https://loopback.io/doc/en/contrib/git-commit-messages.html.

humansonofhuman commented:
@dhmlau it seems like I deleted my fork of the repo at the beginning of the year, so it think I can't amend my commit 😬
But! I can make a new fork and pull request

This commit message is ok?
```
Fix the third example on Remothe-hooks.md

Update the example according to what is said on #1816

While I was working whit an afterRemote Hook I tried to delete a
property of the results following the example in
https://loopback.io/doc/en/lb3/Remote-hooks#common/models/customer.js
(the second one) but it didn't worked for me, so I google it, and
I saw this closed issue strongloop/loopback#1816 and discovered that
I needed to use result.unsetAttribute('property_name') to do it…
so since the example didn't worked out I was thinking on send a pull
request changing that example.

Since I deleted the my fork of the project I remade the pull request
from a new fork.

close #772
```

jannyHou reviewed (approved):
LGTM

jannyHou commented:
@mmgescuelaup Thank you for the contribution, could you fix the commit message by removing the description part?
Thanks. I understand it's always good to keep the background of issue. Writing them down in the issue comment is good enough :) Let's keep the PR description concise.

> While I was working whit an afterRemote Hook I tried to delete a property of the results following the example in https://loopback.io/doc/en/lb3/Remote-hooks#common/models/customer.js (the second one) but it didn't worked for me, so I google it, and I saw this closed issue strongloop/loopback#1816 and discovered that I needed to use result.unsetAttribute('property_name') to do it… so since the example didn't worked out I was thinking on send a pull request changing that example.

This part could be removed to fix your linting message.

humansonofhuman commented:
@jannyHou I can't amend the commit message of the commits on this pull request, since it comes now from a _ unknown repository _ (because I deleted my previous fork, I thought this pull request will never be atended after the time passed 😓)
But I made a new pull request with the correct linting on the commit message 
https://github.com/strongloop/loopback.io/pull/805
